x/examples/outline-connectivity-app/app_mobile/capacitor.config.ts

```diff
@@ -0,0 +1,27 @@
+import { CapacitorConfig } from '@capacitor/cli';
+
+let config: CapacitorConfig = {
+  appId: "org.outline.sdk.connectivity",
```

Comment:
We should use org.getoutline for new apps, to match our domain. Though in the case of example apps perhaps we want something else.
We should perhaps have "example" somewhere in the name to not convey the idea this is an official app or to confuse us later.